internal/controllers/apis/configmap/disable_test_helper.go

```diff
@@ -0,0 +1,28 @@
+package configmap
+
+import (
+	"context"
+	"strconv"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+)
+
+func UpsertDisableConfigMap(ctx context.Context, client ctrlclient.Client, name string, key string, isDisabled bool) error {
```

Comment:
i think it'd be good to either put this in a testing package, or name it `ForTesting` or smth to make it clear this is only for testing